app/controllers/katello/api/v2/content_views_controller.rb

```diff
@@ -81,11 +81,18 @@ def update
                                                            "in the content view version")
     param :major, :number, :desc => N_("Override the major version number"), :required => false
     param :minor, :number, :desc => N_("Override the minor version number"), :required => false
+
+    param :repos_units, Array, :desc => N_("Specify the list of units in each repo"), :required => false
```

Comment:
Can you better describe this param? 

I think you'd want something like:
```
param :repos_units, Array, :required => false do 
  param :rpm_filenames, Array, :required => false, :Desc => N_('some description')
end
```

Reply:
Ah I see, for some reason I thought you couldn't name inner params on arrays (which makes no sense).

Fixing now.